windows/progress-view/ProgressViewView.cpp

```diff
@@ -71,6 +71,9 @@ namespace winrt::progress_view::implementation {
                     this->Background(imgB);
                 }
             }
+            else if (propertyName == "nativeName") {
+                this->Name(to_hstring(propertyValue.AsString()));
+            }
```

Comment:
is this a custom prop? if so see if we can avoid it by using `testID` instead

Reply:
I know I played around with getting testID to work but it wouldn't show up from some reason on the UWP side, so this was my work around. I'll play around with testID some more and see what specific error I was getting!

Reply:
fixed! Looks like I was setting up View props incorrectly